src/test/resources/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors/expectedCGP_testFamilyPriors.vcf

```diff
@@ -0,0 +1,42 @@
+##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes, for each ALT allele, in the same order as listed">
+##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency, for each ALT allele, in the same order as listed">
+##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
+##INFO=<ID=BaseQRankSum,Number=1,Type=Float,Description="Z-score from Wilcoxon rank sum test of Alt Vs. Ref base qualities">
+##INFO=<ID=ClippingRankSum,Number=1,Type=Float,Description="Z-score From Wilcoxon rank sum test of Alt vs. Ref number of hard clipped bases">
+##INFO=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth; some reads may have been filtered">
+##INFO=<ID=DS,Number=0,Type=Flag,Description="Were any of the samples downsampled?">
+##INFO=<ID=FS,Number=1,Type=Float,Description="Phred-scaled p-value using Fisher's exact test to detect strand bias">
+##INFO=<ID=HaplotypeScore,Number=1,Type=Float,Description="Consistency of the site with at most two segregating haplotypes">
+##INFO=<ID=InbreedingCoeff,Number=1,Type=Float,Description="Inbreeding coefficient as estimated from the genotype likelihoods per-sample when compared against the Hardy-Weinberg expectation">
+##INFO=<ID=MLEAC,Number=A,Type=Integer,Description="Maximum likelihood expectation (MLE) for the allele counts (not necessarily the same as the AC), for each ALT allele, in the same order as listed">
+##INFO=<ID=MLEAF,Number=A,Type=Float,Description="Maximum likelihood expectation (MLE) for the allele frequency (not necessarily the same as the AF), for each ALT allele, in the same order as listed">
+##INFO=<ID=MQ,Number=1,Type=Float,Description="RMS Mapping Quality">
+##INFO=<ID=MQ0,Number=1,Type=Integer,Description="Total Mapping Quality Zero Reads">
+##INFO=<ID=MQRankSum,Number=1,Type=Float,Description="Z-score From Wilcoxon rank sum test of Alt vs. Ref read mapping qualities">
+##INFO=<ID=PG,Number=G,Type=Integer,Description="Genotype Likelihood Prior">
+##INFO=<ID=QD,Number=1,Type=Float,Description="Variant Confidence/Quality by Depth">
+##INFO=<ID=ReadPosRankSum,Number=1,Type=Float,Description="Z-score from Wilcoxon rank sum test of Alt vs. Ref read position bias">
+##INFO=<ID=set,Number=1,Type=String,Description="Source VCF for the merged record in CombineVariants">
+##contig=<ID=20,length=63025520,assembly=b37>
+##contig=<ID=21,length=48129895,assembly=b37>
+##reference=human_b37_both.fasta
+##source=CalculateGenotypePosteriors
+##source=SelectVariants
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12878	NA12891	NA12892
+20	564477	.	A	G	0	.	AC=0;AF=0.00;AN=6;DP=32;FS=0.000;MLEAC=0;MLEAF=0.00;MQ0=0;PG=0,0,0;set=Intersection	GT:AD:DP:GQ:JL:JP:PL:PP	0/0:19,0:19:66:9:9:0,57,632:0,66,672	0/0:3,0:3:9:9:9:0,9,109:0,9,164	0/0:10,0:10:30:9:9:0,30,356:0,30,411
```

Comment:
does it matter? the reference is not used AFAIK

On Fri, Jul 22, 2016 at 10:59 AM, ldgauthier <notifications@github.com>
wrote:

> In
> src/test/resources/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors/expectedCGP_testFamilyPriors.vcf
> <https://github.com/broadinstitute/gatk-protected/pull/608#discussion_r71892820>
> :
>
> > +##INFO=<ID=MLEAC,Number=A,Type=Integer,Description="Maximum likelihood expectation (MLE) for the allele counts (not necessarily the same as the AC), for each ALT allele, in the same order as listed">
> > +##INFO=<ID=MLEAF,Number=A,Type=Float,Description="Maximum likelihood expectation (MLE) for the allele frequency (not necessarily the same as the AF), for each ALT allele, in the same order as listed">
> > +##INFO=<ID=MQ,Number=1,Type=Float,Description="RMS Mapping Quality">
> > +##INFO=<ID=MQ0,Number=1,Type=Integer,Description="Total Mapping Quality Zero Reads">
> > +##INFO=<ID=MQRankSum,Number=1,Type=Float,Description="Z-score From Wilcoxon rank sum test of Alt vs. Ref read mapping qualities">
> > +##INFO=<ID=PG,Number=G,Type=Integer,Description="Genotype Likelihood Prior">
> > +##INFO=<ID=QD,Number=1,Type=Float,Description="Variant Confidence/Quality by Depth">
> > +##INFO=<ID=ReadPosRankSum,Number=1,Type=Float,Description="Z-score from Wilcoxon rank sum test of Alt vs. Ref read position bias">
> > +##INFO=<ID=set,Number=1,Type=String,Description="Source VCF for the merged record in CombineVariants">
> > +##contig=<ID=20,length=63025520,assembly=b37>
> > +##contig=<ID=21,length=48129895,assembly=b37>
> > +##reference=human_b37_both.fasta
> > +##source=CalculateGenotypePosteriors
> > +##source=SelectVariants
> > +#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12878	NA12891	NA12892
> > +20	564477	.	A	G	0	.	AC=0;AF=0.00;AN=6;DP=32;FS=0.000;MLEAC=0;MLEAF=0.00;MQ0=0;PG=0,0,0;set=Intersection	GT:AD:DP:GQ:JL:JP:PL:PP	0/0:19,0:19:66:9:9:0,57,632:0,66,672	0/0:3,0:3:9:9:9:0,9,109:0,9,164	0/0:10,0:10:30:9:9:0,30,356:0,30,411
>
> Since you moved these to chromosome 20, are these actually the correct
> reference bases for these positions? I'm skeptical that this VCF is valid.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/broadinstitute/gatk-protected/pull/608/files/9c2b17216b77a6e0bd6d243672ae50d047b7ccaa..e8f19271715c797dfa71cb001a69f382df71e61f#r71892820>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB5rLzCbPvI3Fwi6--AX_dAYJFf3VZBTks5qYNrngaJpZM4JIOHe>
> .
>